app/views/pieces/show.html.erb

```diff
@@ -0,0 +1,21 @@
+<br />
+<h1 class="text-center title"><%= current_game.name %></h1>
+<br />
+<table class="chess_table">
+  <% 7.downto(0).each do |row_number| %>
+    <tr>
+      <% 0.upto(7).each do |column_number| %>
+        <td class="<%= (row_number + column_number).even? ? "black_space" : "white_space"%>">
+          <% piece = current_game.pieces.find_by(x_position: column_number, y_position: row_number) %>
+          <% if piece && current_piece == piece %>
+            <p class='piece highlight'><%= piece.to_unicode %></p>
+          <% elsif piece %>
+            <p class='piece'><%= piece.to_unicode %></p>
+          <% end %>
+        </td>
+      <% end %>
```

Comment:
Next step for this view is: Add a button that connects to `pieces#update` to move the piece.

The button can go on all the filled in squares plus the empty squares.

😄 

(Can do this in a separate pull request if you'd like)